demosplan/DemosPlanCoreBundle/Logic/Segment/SegmentsExporter.php

```diff
@@ -137,6 +159,12 @@ protected function addStatementInfo(Section $section, Statement $statement): voi
         $this->addSegmentCell($row6, $orgaInfoHeader->getNextHeader(), $this->styles['statementInfoTextCell']);
         $this->addSegmentCell($row6, '', $this->styles['statementInfoEmptyCell']);
 
+        if ('' !== $this->getSimilarStatementSubmitters($statement)) {
```

Comment:
You could save the result of the method in a variable to avoid the performance penalty of a duplicate method call

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/2069/commits/bf80b22b99da347027c4caea95e1f31796a23673